src/App.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
+import { HashRouter, Routes, Route } from 'react-router-dom';
 import './App.css';
 import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quote from './components/Quote';
+import Navbar from './components/Navbar';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    return <Calculator />;
-  }
+function App() {
+  return (
+    <HashRouter>
+      <Routes>
+        <Route path="/" element={<Navbar />}>
+          <Route index element={<Home />} />
+          <Route path="calculator" element={<Calculator />} />
+          <Route path="quote" element={<Quote />} />
+        </Route>
+      </Routes>
+    </HashRouter>
```

Comment:
- [ ] [Optional] [Highly Recommended] You only need `Routes` and `Route` from the react-router library for this project and you give a route for every page you need to display in your SPA. So you do not need a route for the navbar because the navbar is a fixed part of every page in your app and you suppose to give `/` path to the `Home` component not the navbar, So please fix that. see [this ](https://github.com/Omar-Muhamad/math-magicians/blob/dev/src/App.js)for better understanding.

[Hint]: 
```react
function App() {
  return (
      <Navbar />
      <Routes>
          <Route path="/" element={<Home />} />
          <Route path="calculator" element={<Calculator />} />
          <Route path="quote" element={<Quote />} />
      </Routes>
  );
}
```

[Notice] I marked that change as optional because your app now is working just fine.